src/editors/Editor.test.jsx

```diff
@@ -12,7 +12,7 @@ jest.mock('./hooks', () => ({
 
 jest.mock('./containers/TextEditor', () => 'TextEditor');
 jest.mock('./containers/VideoEditor', () => 'VideoEditor');
-jest.mock('./containers/ProblemEditor/ProblemEditor', () => 'ProblemEditor');
+jest.mock('./containers/ProblemEditor', () => 'ProblemEditor');
```

Comment:
This fixes is alread coming in https://github.com/open-craft/frontend-lib-content-components/pull/7

Reply:
Thanks! I'm not actively seeking these out, just making changes when they're short